# Limiar de definição entre branco e preto utilizado pela função treshould do openCV
int32 limiarCanny
int32 limiarHough
